src/css/responsive/tablet-landscape.styl

```diff
@@ -47,3 +47,10 @@ $nav-height = 54px
   .grid-item:not(:last-child)
     img.social-icon
       margin-right: $margin-tiny
+
+// ******************************************************
+
+// CART
+
+.icon-remove
+  width: 12px
```

Comment:
does this size relate to type or something? Could it be calculated from that?

Reply:
it doesn't relate to type. i was just eyeing it... 🤷‍♂️